src/main/java/stsjorbsmod/cards/wanderer/DeathThroes.java

```diff
@@ -43,14 +43,18 @@ public DeathThroes() {
     @Override
     public void use(AbstractPlayer p, AbstractMonster m) {
         addToBot(new DamageAction(m, new DamageInfo(p, damage), AttackEffect.FIRE));
-        addToBot(new RememberSpecificMemoryAction(p, WrathMemory.STATIC.ID));
+        if (upgraded) {
+            addToBot(new GainSpecificClarityAction(p, WrathMemory.STATIC.ID));
+        } else {
+            addToBot(new RememberSpecificMemoryAction(p, WrathMemory.STATIC.ID));
+        }
     }
 
     @Override
     public void upgrade() {
         if (!upgraded) {
             upgradeName();
-            upgradeDamage(UPGRADE_PLUS_DAMAGE);
+            tags.remove(REMEMBER_MEMORY);
             upgradeDescription();
```

Comment:
Missing an `UPGRADE_DESCRIPTION` in the loc file?

Reply:
I thought I had done that :(